unity/CITools/BuildDriver/GlobalConfig.cs

```diff
@@ -8,5 +8,6 @@ public class GlobalConfig
     public required string Architecture;
     public required string Configuration;
     public required bool Silent;
+    public required TestTargets TestTargets;
```

Comment:
I'm not a huge fan of adding TestTargets here just for the embedding layer tests. I'd rather we change `Test` to `TestManaged` and then call them back to back based on what flags are set from `Program`

Reply:
Ok.  I'll make that change.